src/components/Carousel/Carousel.svelte

```diff
@@ -65,7 +84,12 @@
   /**
    * Pause autoplay on focus
    */
-  export let pauseOnFocus = false
+  export let pauseOnFocus = true
+
+  /**
+   * Show autoplay duration progress indicator
+   */
+  export let autoplayProgressVisible = false
```

Comment:
Small improvement,
For consistency keep same spelling in docs 'autolplay' or 'auto play'.